invokeai/frontend/web/public/locales/en.json

```
@@ -326,7 +326,8 @@
         "drop": "Drop",
         "dropOrUpload": "$t(gallery.drop) or Upload",
         "dropToUpload": "$t(gallery.drop) to Upload",
-        "deleteImage": "Delete Image",
+        "deleteImage": "Delete single Image",
+        "deleteImage_plural": "Delete {{count}} Images",
```

Don't need "single" here

```suggestion
        "deleteImage": "Delete Image",
        "deleteImage_plural": "Delete {{count}} Images",
```

---

Done. Thanks for the explanation.